build-system/azure-pipeline.template.yaml

```diff
@@ -26,6 +26,12 @@ jobs:
         inputs:
           version: 3.1.x
       # Linux or macOS
+      - bash: |
+          sudo npm install -g azurite@3.17.1
+          sudo mkdir azurite
+          sudo azurite --silent --location azurite --debug azirute/debug.log &
```

Comment:
You wanted to do this over `Docker.NET`?

Reply:
No, the Azurite Docker image rarely got updated and it is linux only. Its the same reason why we switched to this way of doing it in Akka.Persistence.Azure.

Reply:
Got it, thanks